narayana-integration-bom/pom.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.kie</groupId>
+    <artifactId>kie-user-bom-parent</artifactId>
+    <version>7.30.0-SNAPSHOT</version>
+    <relativePath>../kie-user-bom-parent/pom.xml</relativePath>
+  </parent>
+
+  <groupId>org.kie</groupId>
+  <artifactId>narayana-integration-bom</artifactId>
+  <packaging>pom</packaging>
+
+  <name>Narayana Integration BOM (Bill Of Materials)</name>
+  <description>
+    Import this BOM in your dependencyManagement if you want to include narayana in your project.
+  </description>
+
+  <!-- IMPORTANT: Do not declare any build things here! Declare them in kie-user-bom-parent. -->
+  <build />
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.jboss.jbossts.jta</groupId>
+        <artifactId>narayana-jta</artifactId>
+      </dependency>
+
+      <!-- Integration between Tomcat and Narayana -->
+      <dependency>
+        <groupId>org.jboss.integration</groupId>
+        <artifactId>narayana-tomcat</artifactId>
+      </dependency>
+      <!-- TODO: remove this legacy dependency once all repositories switch to narayana-tomcat -->
+      <dependency>
```

Comment:
I just checked the kie-parent, drools, jbpm and droolsjbpm-integration repositories and haven't found this dependency being used anywhere.

If it's not used, let's remove it.

Reply:
@rsynek it is used here
```
$ grep -r ">narayana-tomcat<" ./ --include pom.xml
./jbpm/jbpm-container-test/jbpm-in-container-test/jbpm-container-test-suite/pom.xml:                      <artifactId>narayana-tomcat</artifactId>
./jbpm/jbpm-container-test/jbpm-in-container-test/jbpm-container-test-suite/pom.xml:          <artifactId>narayana-tomcat</artifactId>
./droolsjbpm-build-bootstrap_pme/pom.xml:        <artifactId>narayana-tomcat</artifactId>
./droolsjbpm-build-bootstrap/pom.xml:        <artifactId>narayana-tomcat</artifactId>
./droolsjbpm-integration/kie-server-parent/kie-server-tests/pom.xml:                      <artifactId>narayana-tomcat</artifactId>
./droolsjbpm-integration/kie-server-parent/kie-server-tests/pom.xml:          <artifactId>narayana-tomcat</artifactId>

```

Reply:
`narayana-tomcat` is a successor of `tomcat-jta`. I am calling for removing just the `tomcat-jta`.

Reply:
You are right, it's not used anywhere. Removed.